### DISCUSSION TITLE
unable to change email address

### DISCUSSION THREAD

**Discussion by SeanNZWN** [mastodon/mastodon#16716] [Q&A]
I want to change my email address publicly associated with my mastodon profile. I changed the email address to login in assuming that would do the trick but it has not. I have checked through the settings and searched for a solution but so far nothing. 

Am I missing something or is this a bug?

**Comment by ClearlyClaire** [comment]
I'm confused. The e-mail address associated with your account is never made public, so I'm not sure what you are talking about.

**Comment by Gargron** [comment]
Are you talking about changing your username that kind of looks like an e-mail address, `@username@domain`? That is not possible and it is not an e-mail address, you cannot receive e-mails through it.
